### PR TITLE
Feat/web3 api phase a

### PR DESCRIPTION
**Pull request by amiryadid-fb**
## Pull Request Description

Adding the first functionality of the web3 api (connecting, approving & removing WalletConnect sessions via the api).

## Type of change

Please delete options that are not relevant.

- [ V ] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ V ] Locally tested against Fireblocks API

## Checklist:

- [ V ] I have performed a self-review of my own code
- [ V ] I have made corresponding changes to the documentation
- [ V ] Any dependent changes have been merged and published in downstream modules
- [ V ] I have added corresponding labels to the PR

### CODE REVIEW

**Review by idanya — CHANGES_REQUESTED** [review]
Please remove all `I` prefixes for the new interfaces

File: src/fireblocks-sdk.ts [hunk 1]
```diff
@@ -1146,6 +1149,48 @@ export class FireblocksSDK {
...
+        return await this.apiClient.issuePutRequest(`/v1/connections/${sessionId}`, {approve});
+    }
+
+    /**
+     * Remove an existing connection
+     * @param sessionId The ID of the session
+     */
+    public async removeSignerConnection(sessionId: string): Promise<void> {
+        return await this.apiClient.issueDeleteRequest(`/v1/connections/${sessionId}`);
+    }
```
Comment (idanya) [inline src/fireblocks-sdk.ts:1191]: no response object?

File: src/fireblocks-sdk.ts [hunk 2]
```diff
@@ -1146,6 +1149,48 @@ export class FireblocksSDK {
...
+
+    /**
+     * Approve or Reject the initiated connection
+     * @param sessionId The ID of the session
+     * @param approve Whether you approve the connection or not
+     */
+    public async submitSignerConnection(sessionId: string, approve: boolean): Promise<void> {
+        return await this.apiClient.issuePutRequest(`/v1/connections/${sessionId}`, {approve});
+    }
```
[the remark this answers is not in the document]
Comment (amiryadid-fb) [reply src/fireblocks-sdk.ts:1183]: nope just 200

File: src/fireblocks-sdk.ts (hunk 1 above)
Comment (amiryadid-fb) [reply src/fireblocks-sdk.ts:1191]: nope just 200

**Review by idanya — APPROVED** [review]

**Pull request merged**